support/rollup/examples.js

```diff
@@ -0,0 +1,66 @@
+import path from 'path'
+import alias from 'rollup-plugin-alias'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import json from 'rollup-plugin-json'
+import replace from 'rollup-plugin-replace'
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
+import sourcemaps from 'rollup-plugin-sourcemaps'
+import pkg from '../../package.json'
+
+const configurations = []
+
+const umdConfig = {
+  input: 'examples/index.js',
+  output: {
+    file: pkg.browser,
+    name: 'slate-examples',
+    format: 'umd',
+    exports: 'named',
+    sourcemap: process.env.ROLLUP_WATCH,
+  },
+  plugins: [
+    alias({
+      debug: path.resolve(__dirname, 'node_modules', 'debug', 'src', 'browser'),
```

Comment:
Can you use `'debug'` (quoted) to keep them the same?

Reply:
I don't think the `, , ,` is necessary? Can't you just do `path.resolve(__dirname, 'node_modules/debug/src/browser')` instead? It would be a lot more readable.